### PR TITLE
Hw01

### PR DESCRIPTION
**Pull request by laurenguillot**
I found the GitHub guide for Markdown very useful. After making my changes to README.md, I saved the changes and I will push the changed README.md to GitHub. I recognize that starting out this process is difficult, and I have written notes to help me in the future. I made a step-by-step instruction guide for pulling from GitHub, saving changes, and pushing the commit back to GitHub. I need to get more familiar with the terminology used in coding.

### CODE REVIEW

**Comment by hongjinjo** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**           |     v      |              |            |
| **Coding strategy**        |     v      |              |            |
| **Presentation**           |     v      |              |            |
| **Achievement, creativity** |     v     |             |            |
| **Ease of access**         |     v      |              |            |

Remarks: 

* It seems that Lauren did not put enough # to header (introduction). 
* The introduction was very concise and clear.

**Comment by hannamn** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   x        |              |            |
| **Coding strategy**         |       x    |              |            |
| **Presentation**            |           |    x          |            |
| **Achievement, creativity** |           |    x          |            |
| **Ease of access**          |       x    |              |            |

Remarks:
* Followed the prompt: header, bold text, link - I think the link setup is not perfect because you're missing a ) at the end of the link, but you still managed to make it work so that's great!
* The presentation perhaps could have improved with more headers and if you had cleaned up the original text.
* Your research project sounds great! I

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # hw01
 Repository for homework 01
+#Introduction
```
Comment (mayank-kgp) [inline README.md:3]: Space should have been given between # and Introduction.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
...
+My name is **Lauren Guillot**. I am a student in the *Masters of Arts in the Social Sciences program*. I study political science, specifically [American politics](https://en.wikipedia.org/wiki/American_politics_(political_science) with a focus on public opinion, political behavior, and political psychology.
```
Comment (mayank-kgp) [inline README.md:4]: One more closing bracket should have been there.
`[American politics](https://en.wikipedia.org/wiki/American_politics_(political_science))`

This is a very common mistake. Each opening bracker should have corresponding closing bracket. 
For example - 
`((a+b + c)` is not valid while `((a+b) + c)` is valid.

**Comment by mayank-kgp** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |        x      |            |
| **Coding strategy**         |           |              |     x       |
| **Presentation**            |           |        x     |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |   x       |              |            |

Remarks:

* You can be more careful because minor things like missing bracket or space can be very important in programming. Try to self evaluate your output before final submission.
* Good to see that you have started well in Git workflow. I hope you won't have to see your notes after week 3-4.

**Pull request closed without merging**